doc/common-issues.rst

```diff
@@ -0,0 +1,98 @@
+Frequently Encountered Issues
+=============================
+Also see the `TLS Examples <https://pymongo.readthedocs.io/en/stable/examples/tls.html>`_.
```

Comment:
This should be an RST link to the "Troubleshooting TLS Errors" section. You can add a cross doc link like this:
```

.. _TLSErrors:

Troubleshooting TLS Errors
..........................
```